app/models/conversation.rb

```diff
@@ -5,6 +5,16 @@ class Conversation < ApplicationRecord
 
   scope :active, -> { where(Question.active.where("questions.conversation_id = conversations.id").arel.exists) }
 
+  scope :exclude_opted_out_users, lambda {
+    excluded_ids = Rails.configuration.govuk_chat_private&.opted_out_user_ids || []
```

Comment:
```suggestion
  scope :exclude_opted_out_end_user_ids, lambda {
    excluded_ids = Rails.configuration.govuk_chat_private&.opted_out_end_user_ids || []
```

Just a couple of minor naming suggestions for consistency